### ISSUE TITLE
File 'data/gtfobins.json' missing

### ISSUE THREAD

**Issue by Brazier85** [calebstewart/pwncat#14]
```
(env)   ~/src  python -m pwncat
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/brazier85/src/pwncat/env/lib/python3.8/site-packages/pwncat/__main__.py", line 84, in <module>
    main()
  File "/home/brazier85/src/pwncat/env/lib/python3.8/site-packages/pwncat/__main__.py", line 23, in main
    pwncat.victim = Victim()
  File "/home/brazier85/src/pwncat/env/lib/python3.8/site-packages/pwncat/remote/victim.py", line 125, in __init__
    self.gtfo: GTFOBins = GTFOBins("data/gtfobins.json", self.which)
  File "/home/brazier85/src/pwncat/env/lib/python3.8/site-packages/pwncat/gtfobins.py", line 378, in __init__
    with open(gtfobins, "r") as filp:
FileNotFoundError: [Errno 2] No such file or directory: 'data/gtfobins.json'
```

**Comment by calebstewart** [comment]
This is most likely due to `pwncat` referencing the installation files directly as if it were being run from the source directory. I assume `~/src` is not where you cloned `pwncat`. They should be referenced based on package installation directory through `pkgutils`. I'll take a look at this tonight. Should be pretty simple to fix. This is an oversight on my part due to most of the testing being done from a cloned repository vice a standard `pip install ...` installation.

**Comment by calebstewart** [comment]
I've added the `gtfobins.json` file to the package data in `setup.py`. This should make python setuptools pull the GTFOBins database into the package itself. Also, `pwncat` references the file using `pkg_resources` so the path is correct. I tested this on a kali machine, and it appears to function just fine. I opted to leave the default configuration file and example keys in the same location as before because they don't conceptually need to be installed as part of the package for it to function normally.

Let me know if this works for you!
